cartoframes/viz/map.py

```diff
@@ -339,16 +339,17 @@ def _get_publication_html(self, name):
             layers=_get_layer_defs(self._publisher.get_layers()),
             bounds=self.bounds,
             size=None,
+            camera=self.camera,
```

Comment:
Can we add a test here? I think we fixed that in the past and it'd be great if we check it won't happen again.